pkg/accounting/accounting.go

```diff
@@ -0,0 +1,10 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package accounting
+
+//Update queries and updates nodeID in the accounting DB
+func Update(nodeID string, nodeSize int64) error {
```

Comment:
See my comments in `checker.go`. If I understand what this is doing, I think the name `pieceSize` makes more sense